modules/collectd/templates/etc/collectd/conf.d/elasticsearch-aws.conf.erb

```diff
@@ -0,0 +1,100 @@
+<Plugin curl_json>
```

Comment:
Are all these metrics being reported successfully now?

Reply:
Yes, they were previously being collected as a `counter` (so the values were constantly being added).  I've changed that to `gauge`, which reports just the raw value.